src/Stopwatch_TEST.cc

```diff
@@ -154,8 +155,8 @@ TEST(Stopwatch, StartStopReset)
   watch.Start(true);
   EXPECT_TRUE(watch.Running());
   EXPECT_LT(watch.StopTime(), watch.StartTime());
-  EXPECT_NE(math::clock::duration::zero(), watch.ElapsedRunTime());
-  EXPECT_EQ(math::clock::duration::zero(), watch.ElapsedStopTime());
+  EXPECT_NE(steady_clock::duration::zero(), watch.ElapsedRunTime());
```

Comment:
windows failed on this line; I think it's flaky

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz_math-pr-win&build=93)](https://build.osrfoundation.org/job/gz_math-pr-win/93/) https://build.osrfoundation.org/job/gz_math-pr-win/93/

Reply:
retrying:

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz_math-pr-win&build=95)](https://build.osrfoundation.org/job/gz_math-pr-win/95/) https://build.osrfoundation.org/job/gz_math-pr-win/95/